docs/faq.md

```diff
@@ -110,3 +110,18 @@ Single spa supports code splits. There are so many ways to code split we won't b
 
     * For SystemJS 2-5: Find a code example [here](https://gitlab.com/TheMcMurder/single-spa-portal-example/blob/master/people/src/set-public-path.js#L3)
 1. Set either [`output.jsonpFunction`](https://webpack.js.org/configuration/output/#outputjsonpfunction) or [`output.library`](https://webpack.js.org/configuration/output/#outputlibrary) to ensure that each app's webpack doesn't collide with other apps' webpack. `jsonpFunction` is preferred.
+
+## Does single-spa require additional security considerations?
+
+No. single-spa does not add, deviate, or attempt to bypass any browser JavaScript security measures. The security needs of your applications are the same as if you did not use single-spa.
+
+Outside of that, web applications may use the following resources that have their own security considerations that you may need to become familiar with.
+
+- [ES6 module dynamic imports](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Statements/import)
```

Comment:
The links here don’t really explain security as much as just the topics. Might be nice to link to both the topic and a discussion about its security. If those links are hard to find I’m okay with omitting them, though

Reply:
Yea, I couldn't find anything that delved into security specifically with each of these topics, but I figured these links would at least be where you'd start learning about how they work. 

Maybe this would be a good link for module dynamic imports security? https://github.com/w3c/webappsec-csp/issues/243

Reply:
I updated some of the links to point to the sections that talk about security specifically.